src/www/.storybook/localize.ts

```diff
@@ -25,6 +25,8 @@ type FormattableMessage =
   | FormatXMLElementFn<symbol | object, string | symbol | object | (string | symbol | object)[]>;
 
 export const localize = (messageID: string, ...formatKeyValueList: FormattableMessage[]): string => {
+  if (!messageID) return "";
```

Comment:
This means we will fail silently if messages are missing.
Should we output something else, like "MISSING_MESSAGE"?

We should at least call console.warn

Reply:
Actually, there are instances where a messageID is just not provided. For example, errorMessageId is a required property on `ServerListItem`. If an empty string is given, we'd see `MISSING_MESSAGE` everywhere there's an error, even if there's no message to show.

You know what - I'll just fix the bug at the source